content/docs/alerting/configuration.md

```diff
@@ -301,6 +306,8 @@ to: <tmpl_string>
 
 # The HTML body of the email notification.
 [ html: <tmpl_string> | default = '{{ template "email.default.html" . }}' ]
+# The text body of the email notification.
+[ text: <tmpl_string> ]
```

Comment:
This parameter has been added in https://github.com/prometheus/alertmanager/pull/934.